pom.xml

```diff
@@ -270,19 +259,31 @@
 				</executions>
 			</plugin>
 			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-gpg-plugin</artifactId>
-				<version>3.0.1</version>
-				<executions>
-					<execution>
-						<id>sign-artifacts</id>
-						<phase>verify</phase>
-          				<goals>
-							<goal>sign</goal>
-						</goals>
-					</execution>
-				</executions>
+		        <groupId>org.apache.maven.plugins</groupId>
+		        <artifactId>maven-gpg-plugin</artifactId>
+		        <version>3.0.1</version>
+		        <executions>
+		        	<execution>
+		        		<id>sign-artifacts</id>
+		        		<phase>verify</phase>
+		            	<goals>
+		        			<goal>sign</goal>
+		        		</goals>
+		        	</execution>
+		        </executions>
 			</plugin>
+			<plugin>
+      			<groupId>org.sonatype.plugins</groupId>
+      			<artifactId>nexus-staging-maven-plugin</artifactId>
+      			<version>1.6.7</version>
+      			<extensions>true</extensions>
+      			<configuration>
+        			<serverId>ossrh</serverId>
+        			<nexusUrl>https://s01.oss.sonatype.org/</nexusUrl>
+        			<!-- skipStagingRepositoryClose>true</skipStagingRepositoryClose -->
```

Comment:
are we going to use this on the future or we can safely remove it?

Reply:
It's now disabled :-)